### ISSUE TITLE
TypeError: Cannot read property 'checked' of undefined

### ISSUE THREAD

**Issue by waitingcheung** [kenwheeler/cash#167]
When an object with a property having a value ``undefined`` is passed to the API [``$(element).prop(object)``](https://github.com/kenwheeler/cash#fnprop), a TypeError occurs.

A simple test case to illustrate the TypeError.
```js
$('.prop-fixture').prop({
    "checked": undefined
});
```

The stack trace of the TypeError.
```
{ file:///path/cash.js:305
        return (value === undefined ? this[0][name] : this.each(function (v) {
                                             ^

TypeError: Cannot read property 'checked' of undefined
    at Init.prop (file:///path/cash.js:305:46)
    at Init.prop (file:///path/cash.js:311:14)
```

**Comment by fabiospampinato** [comment]
I think this is happening because your collection doesn't contain any elements. 

Cash throws errors sometimes if the collection is empty. I've fixed this in [my fork](https://github.com/fabiospampinato/cash).

**Comment by fabiospampinato** [comment]
> I think this is happening because your collection doesn't contain any elements.

Fixed in v2.1.2.
